test/e2e/manifests/new_manifests/datadog-agent-ccr-enabled.yaml

```diff
@@ -7,6 +7,7 @@ metadata:
     agent.datadoghq.com/e2e-test: datadog-agent-ccr-enabled
 spec:
   global:
+    clusterAgentToken: thisisa32characterstoken00000000
```

Comment:
There's rarely a race condition I'm investigating where the DCA token is set to an older value (possibly from previous reconciliation or something) while the CCR has the new / correct value (from the secret). This causes the CCR to be unable to connect to DCA and the test fails eventually. By forcing a token, we ensure we don't end up in this situation.